openapi/vendor/oci-schemas/errors.schema.json

```diff
@@ -0,0 +1,26 @@
+{
+   "$id": "http://opencontainers.org/schema/error",
+   "$schema": "http://json-schema.org/draft-07/schema#",
+   "description": "This property contains a list of errors messages returned from the server due to an invalid request.",
+   "properties": {
+      "errors": {
+         "items": {
+            "properties": {
+               "code": {
+                  "type": "string"
```

Comment:
enum?

Reply:
The distribution spec doesn't supply an official jsonschema for errors. Let me double-check but I believe the schema is correct.

Reply:
I agree that it is correct, but it can be even more precise, because the official documentation contains a enumeration of possible error codes.

Reply:
Ah you're right. Totally forgot about that. I'll work on it!

Reply:
Enums added.